package.json

```diff
@@ -36,6 +90,7 @@
     "deep-object-diff": "1.1.0",
     "env-cmd": "10.1.0",
     "graphql": "15.5.0",
+    "graphql-tag": "^2.11.0",
```

Comment:
why is this needed

Reply:
I see that graphql-tag/loader is utilized in webpack.config. maybe this can be a dev dep?

Reply:
This is used to load graphql documents. We use the `loader` function in our webpack build